wiki/Beatmaps/Genre_and_language/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of source, genre, language, and tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
+
+[Beatmaps](/wiki/Beatmaps) submitted to the website have **genre** and **language** fields to help categorise songs. They default to `Unspecified` and `Other` respectively, and can be edited by [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) and [Global Moderation Team](/wiki/People/The_Team/Global_Moderation_Team) members. Mapset owners are also able to set the genre and language of their beatmaps as long as the beatmap's status is WIP<!-- TODO: link -->, [Pending](/wiki/Beatmaps#pending), or [Graveyard](/wiki/Beatmaps#graveyard).
+
+Genre and language have no function in the game, but will be available as search options in osu!lazer<!-- TODO: link me -->.
+
+## Beatmap listing
+
+![The available filters on the beatmap listing](img/beatmap-listing.png)
+
+The [beatmap listing](https://osu.ppy.sh/beatmapsets) has filters available for genre and language in the `More Search Options` dropdown.
+
+## List of genres
+
+| Genre | Traits of applicable songs |
```

Comment:
Think a more natural order could be used for this list, regular users can't see the dropdown for genres.
Would move non-genres to the bottom and keep Unspecified + Other, Anime + Video Game next to each other.

Reply:
Eh, it's the same order that's used on the website for the search filter, so the current way seems fitting enough

Reply:
not changed^